### PR TITLE
[PKG-2823] pre-commit 3.4.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/pre-commit/pre-commit/blob/v3.4.0/CHANGELOG.md
License: https://github.com/pre-commit/pre-commit/blob/v3.4.0/LICENSE
Requirements: https://github.com/pre-commit/pre-commit/blob/v3.4.0/setup.cfg

Actions:
1. Reset the build number to `0`

**Current info**

| Name | Downloads | Version | Platforms | License |
| --- | --- | --- | --- | --- |
| PyPi | --- |  PyPI - Version Python versions | --- |  PyPi |
| GitHub | --- | GitHub | --- | GitHub |

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
Rebuilding with `cfgv 3.4.0` https://github.com/AnacondaRecipes/cfgv-feedstock/pull/2

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
   version: {{ version }}
 
 source:
   url: https://github.com/pre-commit/pre-commit/archive/v{{ version }}.tar.gz
-  sha256: 5cc587e824a9dde28e2668f3da547da10607f7e531bdf8a6c1e8a9bdc7134570
+  sha256: 65d63b57ddec920b09ffdf09536a480d6883eb5069eb295daab6d3fe151e34d3
 
 build:
-  number: 0
+  number: 1
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  number: 0
```

**Review by cbouss — APPROVED** [review]
LGTM, with the build number reset

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**